M2/Macaulay2/packages/CMakeLists.txt

```diff
@@ -37,6 +37,8 @@ set(gbTrace                     "0" CACHE STRING "set the Groebner basis trace l
 ## Options to installPackage
 set(CheckDocumentation       "true" CACHE STRING "check documentation for completeness")
 set(IgnoreExampleErrors     "false" CACHE STRING "ignore errors in example code")
+set(MakeDocumentation       "true"  CACHE STRING "make the documentation")
+set(ENV{MakeDocumentation}  ${MakeDocumentation}) # for doc-links.m2
```

Comment:
This is not going to work. The CMake command sets the environment, but once the CMake command ends, the environment clears. Ninja runs M2, not CMake.

Reply:
Ah, good to know.  Maybe a better fix would be to change `doc-links.m2` to just see if the raw documentation exists, and if not, then end there, rather than looking for some environment variable.